config.json

```diff
@@ -1050,6 +1050,18 @@
       "topics": null,
       "deprecated": true
     },
+    {
+      "uuid": "34738b66-28bb-41bc-85c6-68fcf4a09bc2",
+      "slug": "linked-list",
+      "core": false,
```

Comment:
One of the following must be true: `core` is not false, or `unlocked_by` is not null. Travis won't pass otherwise. My instinct is that this should be a core exercise toward the end of the track: everyone could benefit from some `unsafe` practice.

Reply:
I still fear we'd overload the mentors with such an exercise. It's already long and unsafe should get extra care. There's smaller stuff to introduce some unsafe, like, say, implementing a version of `std::vec::Drain`.
I've set it to be unlocked by forth for now, but that's just for a travis test run.